challenges/classes.js

```diff
@@ -1,11 +1,20 @@
 // Copy and paste your prototype in here and refactor into class syntax.
+let CuboidMaker = {
+  constructor(properties) {
```

Comment:
* Consider refactoring this line using the JS Class Syntax like so: ```class CuboidMaker {```
* Then adding a new line bellow for the constructor to accept a properties object like so: 
```constructor( properties ) {```